sidekick/lib/src/init/init_command.dart

```diff
@@ -284,18 +307,32 @@ Future<File> installFlutterWrapper(Directory directory) async {
     r'sh -c "$(curl -fsSL https://raw.githubusercontent.com/passsy/flutter_wrapper/master/install.sh)"',
     workingDirectory: directory,
   );
-  return directory.file('flutterw');
+  final exe = directory.file('flutterw');
+  assert(exe.existsSync());
+  return exe;
+}
+
+/// Downloads the bundled dart runtime for the CLI
+void downloadDartRuntime(Directory sidekickCliPackage) {
+  dcli.run(
+    'sh tool/download_dart.sh',
+    workingDirectory: sidekickCliPackage.path,
+  );
+  final downloadPath = sidekickCliPackage.directory('build/cache/dart-sdk/');
+  while (!downloadPath.existsSync()) {
```

Comment:
Why is `while (!downloadPath.existsSync())` necessary? Shouldn't the folder always exist after `download_dart.sh` has finished executing`?

Reply:
Good catch. This is a leftover from a debugging session